lib/lhm/chunker.rb

```diff
@@ -139,8 +139,6 @@ def execute_batch_mode
         print '.'
       end
       print "\n"
-    rescue => e
-      puts e.message
```

Comment:
is it enough to catch the exception or do we need to raise them to parent calling method?